lib/timeline/component/ItemSet.js

```diff
@@ -258,6 +258,48 @@ ItemSet.prototype._create = function(){
   this.body.dom.centerContainer.addEventListener('mousemove', this._onMouseMove.bind(this));
   // right-click on timeline 
   this.body.dom.centerContainer.addEventListener('contextmenu', this._onDragEnd.bind(this));
+  this.body.dom.centerContainer.addEventListener('mouseout', this._onMouseOut.bind(this));
+
+  this.dragEvent = null;
+
+  this.mouseLocation = {mouseOnLeft: false, mouseOnRight: false, mouseIsLeaving: false}
```

Comment:
I think you should not add another property referring to mouse properties. There is a ` this.touchParams` which contains all the mouse position props. I'd attach these properties to ` this.touchParams` instead of defining `this.mouseLocation`